packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -409,10 +409,70 @@ class FlutterDriver {
 
   /// Scrolls the Scrollable ancestor of the widget located by [finder]
   /// until the widget is completely visible.
+  ///
+  /// If the widget located by [finder] is contained by a scrolling widget
+  /// that lazily creates its children, like [ListView] or [CustomScrollView],
+  /// then this method may fail because [finder] doesn't actually exist.
+  /// The [scrollUntilVisible] method can be used in this case.
   Future<Null> scrollIntoView(SerializableFinder finder, { double alignment: 0.0, Duration timeout }) async {
     return await _sendCommand(new ScrollIntoView(finder, alignment: alignment, timeout: timeout)).then((Map<String, dynamic> _) => null);
   }
 
+  /// If widget located by [item] is not visible then repeatedly [scroll] the
```

Comment:
It's strange to begin a doc comment with a special condition. I'd just start with "Repeatedly"

Reply:
Good point!